src/proto/mgmt/pool.proto

```diff
@@ -70,6 +70,23 @@ message PoolExcludeResp {
 	int32 status = 1; // DAOS error code
 }
 
+// PoolExtendReq supplies pool identifier and rank list.
+message PoolExtendReq {
+	string uuid = 1; // uuid of pool to add target up to
+	repeated uint32 ranks = 2; // ranks
+	uint64 scmbytes = 3; // SCM size in bytes
+	uint64 nvmebytes = 4; // NVMe size in bytes
+	string user = 5; // formatted user e.g. "bob@"
+	string usergroup = 6; // formatted group e.g. "builders@"
+	string sys = 7; // DAOS system identifier
+	repeated string acl = 8; // Access Control Entries in short string format
```

Comment:
Surely we don't want to duplicate these in the extend request...

Reply:
Not permanently, only while we don't have the necessary server side support to fetch everything we need. See above.